vortexasdk/client.py

```diff
@@ -195,6 +197,23 @@ def create_client() -> VortexaClient:
         raise KeyError(
             "VORTEXA_API_KEY environment variable is required to use the VortexaSDK"
         )
+    latest_version, _ = get_latest_package_version(pkg_name)
+    try:
+        version_mask = StrictVersion(__version__) < StrictVersion(
+            latest_version
+        )
+    except ValueError as e:
+        if "invalid version number" in str(e):
+            version_mask = LooseVersion(__version__) < LooseVersion(
```

Comment:
I don't think the variable name `version_mask` is a mask, it's more a boolean representing `sdk_version_is_out_of_date`.

Given that our public versions all follow semver, I think we're safe to just use the `LooseVersion` equality check (and not need the `StrictVersion` try catch).